### DISCUSSION TITLE
Variable tile/ card sizes (showing the full photo) for a more customizable viewing or for building a mood board

### DISCUSSION THREAD

**Discussion by cashewpillar** [photoprism/photoprism#2380] [Ideas]
There are three available views: Card, Mosaic, and List

I am looking for an app that allows me to organize photos in a sort of a mood board and was hoping to see something like below:
![tablet](https://user-images.githubusercontent.com/65220389/170911838-24a1ac8f-3e85-460c-a1bb-059a1b46bb82.jpg)
where the photos aren't cropped to 1x1 boxes, and are shown in varying sizes in the board depending on their quality or some other criteria.

Is there going to be a feature like this? or can anyone let me know of an app with similar viewing features as Google Photos but are more customizable and has the same AI-powered features as Photoprism?

**Comment by graciousgrey** [comment]
We have some related feature requests, feel free to leave a :+1: in there: 

https://github.com/photoprism/photoprism/issues/85

https://github.com/photoprism/photoprism/issues/315
